.idea/workspace.xml

```diff
@@ -9,7 +9,10 @@
     </enabled-run-configurations>
```

Comment:
should .idea/ be added to .gitignore?

Reply:
Thanks! Yup, bad rebase after a Mac rebuild meant I was including pointless config files 🤦‍♂️  rebased on main now :+1: